docs/github-connector/configuration.md

```diff
@@ -0,0 +1,26 @@
+# Configuring GitHub Connector
+
+## Overview
+
+This document describes how to correctly connect GitHub repository to the GitHub Connector installed in Kyma environment. After completion you will be able to handle events incoming from GitHub in lambdas.
```

Comment:
As per guidelines, don't use future tense. See [here](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#tenses).